activestorage/lib/active_storage/attached/changes/create_one.rb

```diff
@@ -61,8 +61,9 @@ def find_or_build_blob
             service_name: attachment_service_name
           )
         when Hash
-          args = attachable.reverse_merge(record: record, service_name: attachment_service_name)
-          ActiveStorage::Blob.build_after_unfurling(**args)
+          ActiveStorage::Blob.build_after_unfurling(
+            attachable.reverse_merge(record: record, service_name: attachment_service_name)
```

Comment:
`**` is necessary to suppress the warning for kwargs splitting.

```suggestion
            **attachable.reverse_merge(record: record, service_name: attachment_service_name)
```

https://buildkite.com/rails/rails/builds/64087#88b1acc6-3973-4109-a9fb-5e34185c08fa/982-1196